corehq/project_limits/tests/test_gauge.py

```diff
@@ -0,0 +1,212 @@
+from unittest.mock import patch
+
+from django.core.cache import cache
+from django.test import SimpleTestCase, TestCase
+
+from nose.tools import nottest
+
+from corehq.apps.change_feed import topics
+from corehq.apps.change_feed.topics import CASE_SQL
+from corehq.project_limits.gauge import (
+    Gauge,
+    PillowLagGaugeLimiter,
+    case_pillow_lag_gauge,
+    get_pillow_throttle_definition,
+)
+from corehq.project_limits.models import PillowLagGaugeDefinition
+
+
+class TestKafkaPillowGauge(SimpleTestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.feature_key = CASE_SQL
+        cls.scopes = topics.get_all_kafka_partitons_for_topic(cls.feature_key)
+        cls.case_gauge = case_pillow_lag_gauge
```

Comment:
Is there a particular reason to save the gauge as a class variable?

Reply:
I don't think so 🤔 I guess I had added it before refactoring where this gauge was used by all the tests.